torchtext/models/__init__.py

```diff
@@ -1 +1,2 @@
 from .roberta import *  # noqa: F401, F403
+from .t5 import *  # noqa: F401, F403
```

Comment:
t5 not added yet so this will cause errors

Reply:
It's added in this commit

Reply:
OH nvm u right